bin/mage

```diff
@@ -18,6 +18,8 @@ define('MAGALLANES_DIRECTORY', $baseDir);
 
 if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
     require_once __DIR__ . '/../vendor/autoload.php';
+} elseif (file_exists(__DIR__ . '/../../../autoload.php')) {
+    require_once __DIR__ . '/../../../autoload.php';
```

Comment:
@SenseException It's partly our fault, because these lines are in `master` branch but has not been  merged into `develop`. See 4cce76472e6ee67d6a88d1abe36c39f403aca7a1
I'll try to merge it locally and see if it won't crash anything (it shouldn't though)